cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/VirtualMachineConfigurationDTO.java

```diff
@@ -49,21 +52,35 @@
     private OSType osType;
 
     // region Auto-generated code
+    public Set<UrlEntityDTO> getInstanceTypes()
+    {
+        return instanceTypes;
+    }
+
+    public void setInstanceTypes( Set<UrlEntityDTO> instanceTypes )
+    {
+        this.instanceTypes = instanceTypes;
+    }
+
+    @Deprecated
     public BigDecimal getCpu()
     {
         return cpu;
     }
 
+    @Deprecated
     public void setCpu( BigDecimal cpu )
     {
         this.cpu = cpu;
     }
 
+    @Deprecated
     public Integer getRam()
     {
         return ram;
     }
 
+    @Deprecated
     public void setRam( Integer ram )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)